presto-spi/src/main/java/io/prestosql/spi/connector/ColumnMetadata.java

```diff
@@ -90,11 +113,13 @@ public boolean isNullable()
         return nullable;
     }
 
+    @Nullable
```

Comment:
Maybe let's log an issue to replace this with Optional.

Reply:
_replacing_ is hard to do in backward compat way

Reply:
Right, at some point there will be backward incompatible changes. On the other hand we can't just leave it to rust and rust.

Reply:
Sure.

Reply:
i added a TODO